int64 aim_num
---
int64 is_back_home